README.md

```diff
@@ -3,39 +3,100 @@
 ## Installation
 1. Clone repository into workspace [It doesn't have to be called moveit_simple_ws]
 ```
-mkdir moveit_simple_ws && cd moveit_simple_ws
-mkdir src && cd src
-catkin_init_workspace
+export CATKIN_WS=$HOME/moveit_simple_ws/
+mkdir -p $CATKIN_WS/src && cd $CATKIN_WS/src
 git clone -b kinetic-devel https://github.com/plusone-robotics/moveit_simple.git
 ```
 
-2. Source Dependencies
+2. Get Source Dependencies
 ```
+cd $CATKIN_WS/src
```

Comment:
In my mind, having these `cd <path>` statements inhibit flow without a tangible benefit. Most people are going to copy and paste these instructions. If the desire here is to enforce you're in a specific directory when the command is run, you can make a compound instruction of `cd <path> && cmd`, but I think that too is unnecessary.

Reply:
I don't see how these would inhibit flow as the new commands are even more copy-paste-able than the existing bash commands. The new README asks the user to set the CATKIN_WS environment variable whenever they `source devel/setup.bash` ensuring that whenever any of these commands are run, they get run in the correct context. 

I added the `&&`'s based on your request